geoportal/c2cgeoportal_geoportal/scaffolds/update/geoportal/CONST_vars.yaml_tmpl

```diff
@@ -100,6 +100,9 @@ vars:
                 radius2: 0
                 angle: 0
         ngeoProfileOptions: {}
+        ngeoStreetviewOptions: 
+          viewer: 'google'
+          key: ''
```

Comment:
The default viewer shouldn't me Mapillary @ybolognini  what do you think?
and the key should be in the `vars.yaml_tmpl` file.